src/components/dialog-full-screen/content-classic.style.js

```diff
@@ -1,12 +1,23 @@
 import { css } from 'styled-components';
 import { THEMES } from '../../style/themes';
 
-export default ({ theme }) => theme.name === THEMES.classic && css`
+export default ({ theme, hasHeader }) => theme.name === THEMES.classic && css`
```

Comment:
Could you use `isClassic` function here please?

Reply:
updated.